aspose/src/main/java/org/frankframework/extensions/aspose/services/conv/impl/convertors/PdfStandaardConvertor.java

```diff
@@ -51,6 +57,15 @@ public void convert(MediaType mediaType, Message message, CisConversionResult re
  		result.setNumberOfPages(getNumberOfPages(result.getPdfResultFile()));
 	}
 
+	private boolean hasSignature(Message message, String charset) throws IOException {
+		try (InputStream is = message.asInputStream(charset)) {
+			try (PdfFileSignature pdfFileSignature = new PdfFileSignature()) {
+				pdfFileSignature.bindPdf(is);
+				return !pdfFileSignature.getSignNames().isEmpty();
+			}
+		}
```

Comment:
Moet je hier niet de BOMInputStream gebruiken, net als bij de "normale" manier om te openen?

Reply:
Tja, het hebben van die bom is sowieso niet goed, eigenlijk moet het zetten van de charset er uit, wat het is allemaal binaire data. daar zit geen bom in en dat heeft geen charset.